.github/dependabot.yml

```diff
@@ -3,4 +3,5 @@ updates:
   - package-ecosystem: "bundler"
     directory: "/"
     schedule:
-      interval: "weekly"
+      interval: "daily"
+    open-pull-requests-limit: 10
```

Comment:
I'm not really sold on this change for 2 reasons:
1. I don't want to be flooded by upgrade PRs in the `rails_api_base`. If we want to make faster progress nothing impedes us to manually go and update stuff.
2. I don't want this change to reach actual projects that are based on the `rails_api_base`

I think 5 PRs/week is a good tradeoff

Reply:
I agree, specially on point 2.